geopandas/io/sql.py

```diff
@@ -92,8 +96,11 @@ def load_geom_text(x):
             srid = shapely.get_srid(geoms.iat[0])
             # if no defined SRID in geodatabase, returns SRID of 0
             if srid != 0:
-                crs = "epsg:{}".format(srid)
-
+                if spatial_ref_sys_df is not None:
+                    entry = spatial_ref_sys_df.loc[spatial_ref_sys_df["srid"] == srid]
+                    crs = f"{entry['auth_name'].values[0]}:{srid}"
```

Comment:
```suggestion
                    crs = f"{entry['auth_name'].item()}:{srid}"
```
I would expect this should be a unique match (although perhaps if there are somehow multiple, falling back to the first is okay)